main.go

```diff
@@ -143,70 +143,43 @@ func main() {
 
 	a := app.New(natsClient, dbconn, userSuffix)
 
-	if err = natsClient.Subscribe(qmssubs.GetUserUpdates, a.GetUserUpdatesHandler); err != nil {
-		log.Fatal(err)
-	}
-
-	if err = natsClient.Subscribe(qmssubs.AddUserUpdate, a.AddUserUpdateHandler); err != nil {
-		log.Fatal(err)
-	}
-
-	// Only call these two endpoints if you need to correct a usage value and
-	// bypass the updates tables.
-	if err = natsClient.Subscribe(qmssubs.GetUserUsages, a.GetUsagesHandler); err != nil {
-		log.Fatal(err)
-	}
-
-	if err = natsClient.Subscribe(qmssubs.AddUserUsages, a.AddUsageHandler); err != nil {
-		log.Fatal(err)
-	}
+	handlers := map[string]nats.Handler{
+		qmssubs.GetUserUpdates: a.GetUserUpdatesHandler,
+		qmssubs.AddUserUpdate:  a.AddUserUpdateHandler,
 
-	// These will get used by frontend calls to check for user overages.
-	if err = natsClient.Subscribe(qmssubs.GetUserOverages, a.GetUserOverages); err != nil {
-		log.Fatal(err)
-	}
+		// Only call these two endpoints if you need to correct a usage value and
+		// bypass the updates tables.
+		qmssubs.GetUserUsages: a.GetUsagesHandler,
+		qmssubs.AddUserUsages: a.AddUsageHandler,
 
-	if err = natsClient.Subscribe(qmssubs.CheckUserOverages, a.CheckUserOverages); err != nil {
-		log.Fatal(err)
-	}
+		// These will get used by frontend calls to check for user overages.
+		qmssubs.GetUserOverages:   a.GetUserOverages,
+		qmssubs.CheckUserOverages: a.CheckUserOverages,
 
-	if err = natsClient.Subscribe(qmssubs.UserSummary, a.GetUserSummaryHandler); err != nil {
-		log.Fatal(err)
+		qmssubs.UserSummary:         a.GetUserSummaryHandler,
+		qmssubs.AddUser:             a.AddUserHandler,
+		qmssubs.GetSubscription:     a.GetSubscriptionHandler,
+		qmssubs.AddQuota:            a.AddQuotaHandler,
+		qmssubs.ListPlans:           a.AddQuotaHandler,
```

Comment:
This line looks wrong -- should be `a.ListPlansHandler` I think?

Reply:
Yup, that's a bug. Thanks!